Newsletters/Yearn Finance Newsletter #45/Yearn Finance Newsletter #45.md

```diff
@@ -1,66 +1,66 @@
-- Ecosystem News
+- Los proveedores de liquidez de ALPHA y YFI serán elegibles para un NFT especial
+- Recordar migrar a V2
+- Como hacer SWAP para entrar a las vaults de Yearn
+- Los ingresos acumulados del protocolo alcanzan los 39M de dólares
+- Las vaults en Yearn
+- Noticias del ecosistema
 
-## **ALPHA and YFI Liquidity Providers Eligible For A Special NFT**
+## **Los proveedores de liquidez de ALPHA y YFI serán elegibles para un NFT especial**
 
 ![](image2.jfif)
 
-Liquidity providers for the Uniswap ETH/YFI and Sushiswap ETH/YFI pools in Alpha Homora V2 pools will be eligible for a special NFT drop courtesy of Yearn, Alpha, and Project Galaxy.
+Proveedores de los fondos de liquidez de ETH/FYI en Alpha Homora V2 dentro de Uniswap y Sushiswap serán elegibles para el envío de un NFT especial, cortesía de Yearn, Alpha y Project Galaxy.
 
-This drop will last 3 full days, from 9/3-9/9. Users that open/have existing positions in eligible pools will be able to claim their NFT on Project Galaxy's website [here](https://galaxy.eco/AlphaFinanceLab/campaign/117).
+Este envío se realizará en el transcurso de 3 días completos, en el periodo de 03/09-09/09. Los usuarios que tengan o abran posiciones en los fondos elegibles podrán hacerse con el NFT en la web de Project Galaxy [Aquí](https://galaxy.eco/AlphaFinanceLab/campaign/117).
 
-Note that users will only be able to claim their NFT during the window and must claim with the wallet address that has the positions currently open
+Los usuarios solo podrán obtener el NFT durante la ventana mencionada y deberán hacerlo con la misma cartera con la que han abierto las posiciones.
 
-View more details [here](https://twitter.com/AlphaFinanceLab/status/1433689307152195591).
+ver mas detalles [Aquí](https://twitter.com/AlphaFinanceLab/status/1433689307152195591).
 
-## **Reminder To Migrate To V2**
+## **Recordar migrar a V2**
 
-As v2 has been live for months, the v1 vaults have been deprecated and strategies have been shut down, with users still in the vaults earning no yield. These vaults have served the DeFi community over a year as a reliable source of yield, and now, it's time for an upgrade. As a reminder, our v2 vaults entail several changes such as deploying capital across several strategies for more efficiency and better risk-adjusted yield. You can migrate to v2 at any time without fees.
+La V2 está activa hace meses , es por esto que las vaults de V1 se han quedado obsoletas se han desactivado las estrategias por lo que los usuarios en estas vaults no obtendrán rendimiento. Estas vaults han servido a la comunidad Defi por más de un año como una fuente confiable de rendimiento, y ahora es el momento para hacer una mejora. Como recordatorio, nuestras vaults V2 contienen diversos cambios como la capacidad de desplegar capital a través de diversas estrategias para mayor eficiencia y mejor ajuste de la relación riego-rendimiento. Puedes migrar a V2 en cualquier momento sin comisiones.
```

Comment:
AS-IS: La V2 está activa hace meses , es por esto que las vaults de V1 se han quedado obsoletas se han desactivado las estrategias por lo que los usuarios en estas vaults no obtendrán rendimiento.
TO-BE: Las V2 vaults están activas desde hace meses, es por esto que las vaults V1 se han quedado obsoletas; se han desactivado las estrategias por lo que los usuarios en estas vaults no obtendrán rendimiento.

AS-IS: ...relación riego-rendimiento.
TO-BE: ...relación riesgo-rendimiento.